### PR TITLE
Remove Gitter Chat Link and Badge from Readme

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Remove Gitter Chat Link and Badge from Readme

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by MohitGupta121** [comment]
@seanlip PTAL, as we discuss in onboarding-space. Gitter-Chat link broken.

**Review by MohitGupta121 — COMMENTED** [review]
@seanlip PTAL, as we discuss in onboarding-space. Gitter-Chat link broken.

File: .github/README.md [hunk 1]
```diff
@@ -67,8 +67,6 @@ The Oppia Android code is released under the [Apache v2 license](https://github.
   * [Discussion forum](http://groups.google.com/group/oppia)
```
Comment (seanlip) [inline .github/README.md]: Could you please change this to the android github discussions page?

**Review by seanlip — COMMENTED** [review]
Thanks @MohitGupta121! One comment.

File: .github/README.md (hunk 1 above)
Comment (MohitGupta121) [reply .github/README.md]: Yes sure.

Comment (MohitGupta121) [reply .github/README.md]: Done

**Review by seanlip — APPROVED** [review]
Thanks! LGTM.

**Review by BenHenning — APPROVED** [review]
Thanks @MohitGupta121.

File: .github/README.md (hunk 1 above)
Comment (BenHenning) [reply .github/README.md]: This looks to be resolved now.

**Pull request merged**